### ISSUE TITLE
design: new icon / logo for extension

### ISSUE THREAD

**Issue by cdrani** [chorus-studio/chorus#206]
I would love an icon that better reflects the extension and it's core functionalities. The current one is ai-generated wordart-esque. 

If you have ideas/suggestions or can design a better one, feel free. Best submission wins.

**Comment by abhi316** [comment]
may i help you

**Comment by cdrani** [comment]
@abhi316: Yes, of course. Whether you design it yourself or make use of ai doesn't really matter. I just want something that can better resemble/describe this extension's purpose and/or uses. Currently I am just using Spotify's brand colours in a lot of the ui, but I can update it to match icon if new colours are introduced instead. Feel free to drop designs below.

**Comment by cdrani** [comment]
New icon in v2. It's not significantly better, but it's an improvement, so there's that.
